src/axom/primal/tests/primal_numeric_array.cpp

```diff
@@ -57,6 +57,28 @@ TEST(primal_numeric_array, constructors)
     CoordType expVal = i < halfPt ? valsArr[i] : CoordType();
     EXPECT_EQ(arrHalfVec[i], expVal);
   }
+
+  primal::NumericArray<int, 3> fromInitializerListRightSize = {10, 20, 30};
```

Comment:
It might be helpful to also have some examples of the form
```cpp
   primal::NumericArray<int, 3> fromInitializerList{10, 20, 30}; 
```

Or perhaps:
```cpp
   primal::NumericArray<int, 3> fromInitializerList{{10, 20, 30}}; 
```

Reply:
Good idea. I will add tests that do this.